Security-Operations/README.md

```diff
@@ -0,0 +1,31 @@
+# Security Operations Roles
+
+Words that resonate with SecOps: Awareness + Response + Prevention. We work proactively and reactively to increase awareness of threats. We are the first responders for security incidents and work closely with our Legal, Compliance, and IT Operations teams. We do the heavy lifting to implement and improve the controls underpinning our Information Security Management System (ISMS) - think of them as guard rails making it easier for teams to do their jobs safely. We have two disciplines/sub-teams: Security Analysts and Security Engineers.
+
+Our roles are defined by expectations for each level in terms of:
+
+- **Planning horizon**: The typical timeframe to be thinking ahead about your impacts
+- **Impact radius**: The number of people or teams typically impacted by your work
+- **Evaluated by**: The person typically accountable for evaluating your performance
+- **Responsibility and direction needed**: The expected degree of autonomy at that level
+
+...and in each of the following dimensions:
+
+- 🦉 Domain Expertise
+- 🌱 Teaching and Mentoring
+- 🧭 Culture and Leadership
+- 🏆 Business Success
+
+The following pages provide demonstrable values and concrete examples to describe these expectations:
```

Comment:
I found myself wanting to understand how the roles are (or are not) related. For example:
1. Is there a separate career track for analysts and engineers?
2. Is "Cloud" a special role?
3. How does the technical lead fit into the picture?

Consider explaining the model that's in your mind.